### PR TITLE
Fix incorrect parameter mapping in Get-AzApplyUpdate

### PR DESCRIPTION
**Pull request by SethuramS**
## Description

Fixed incorrect parameter mapping in Get-AzApplyUpdate

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by SethuramS** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: src/Maintenance/Maintenance.Test/ScenarioTests/MaintenanceTests.ps1 [hunk 1]
```diff
@@ -662,6 +662,97 @@ function Test-AzApplyUpdateCancelConfiguration
     }
  }
 
+ <#
+.SYNOPSIS
+Test New-AzMaintenanceConfiguration, New-AzConfigurationAssignment, Get-AzApplyUpdate
+#>
+function Test-GetAzApplyUpdateWithParentResource
```
Comment (KalpeshChavan12) [inline src/Maintenance/Maintenance.Test/ScenarioTests/MaintenanceTests.ps1:669]: Please add UT for VM applyupdate also

Comment (SethuramS) [reply src/Maintenance/Maintenance.Test/ScenarioTests/MaintenanceTests.ps1:669]: Added test for VM

File: src/Maintenance/Maintenance.Test/Maintenance.Test.csproj [hunk 2]
```diff
@@ -14,6 +14,10 @@
     <Folder Include="SessionRecords\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Azure.ResourceManager.Compute" Version="1.6.0" />
```
Comment (VeryEarly) [inline src/Maintenance/Maintenance.Test/Maintenance.Test.csproj:18]: we do not support track2 sdk, please use track1 sdk instead, also add reference for network sdk

**Comment by VeryEarly** [comment]
please add reference for https://github.com/SethuramS/azure-powershell/blob/main/src/Compute/Compute/Compute.csproj
and https://github.com/SethuramS/azure-powershell/blob/main/src/Network/Network/Network.csproj
in https://github.com/SethuramS/azure-powershell/blob/main/src/Maintenance/Maintenance.sln

File: src/Maintenance/Maintenance.Test/Maintenance.Test.csproj (hunk 2 above)
Comment (SethuramS) [reply src/Maintenance/Maintenance.Test/Maintenance.Test.csproj]: Yes, I was little confused while adding module dependencies. I have now fixed it.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**